cdc/sinkv2/tablesink/progress_tracker.go

```diff
@@ -82,7 +84,7 @@ type progressTracker struct {
 	// The position that the next event which should be check in `advance`.
 	nextToResolvePos uint64
 
-	resolvedTsCache []pendingResolvedTs
+	resolvedTsCache []*pendingResolvedTs
```

Comment:
Why pointer? `pendingResolvedTs` is a small struct, less than 64 bytes. It will increase memory fragmentation.

Reply:
Reverted!